### PR TITLE
Update recover from ransomware

### PR DESCRIPTION
**Pull request by drew-russell**
Update the recover from ransomware documentation to add more wholistic best practices for backup and recovery data protection and better highlight today's M365 threat landscape

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/recover-from-ransomware.md [hunk 1]
```diff
@@ -27,7 +27,8 @@ ms.prod: m365-security
 - [Microsoft Defender for Office 365 plan 1 and plan 2](defender-for-office-365.md)
 - [Microsoft 365 Defender](../defender/microsoft-365-defender.md)
 
-Even if you take every precaution to protect your organization, you can still fall victim to a [ransomware](/windows/security/threat-protection/intelligence/ransomware-malware) attack. Ransomware is big business, and the attacks are very sophisticated.
+Even if you take every precaution to protect your organization, you can still fall victim to a [ransomware](/windows/security/threat-protection/intelligence/ransomware-malware) attack. Ransomware is big business, and in today's threat landscape Microsoft 365 is an ever increasing [target for sophisticated attacks](https://i.blackhat.com/USA21/Wednesday-Handouts/us-21-Cloudy-With-A-Chance-Of-APT-Novel-Microsoft-365-Attacks-In-The-Wild.pdf).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/recover-from-ransomware.md]:
```suggestion
Even if you take every precaution to protect your organization, you can still fall victim to a [ransomware](/windows/security/threat-protection/intelligence/ransomware-malware) attack. Ransomware is big business, and in today's threat landscape Microsoft 365 is an ever-increasing [target for sophisticated attacks](https://i.blackhat.com/USA21/Wednesday-Handouts/us-21-Cloudy-With-A-Chance-Of-APT-Novel-Microsoft-365-Attacks-In-The-Wild.pdf).
```

File: microsoft-365/security/office-365-security/recover-from-ransomware.md [hunk 2]
```diff
@@ -39,9 +40,13 @@ The steps in this article will give you the best chance to recover data and stop
 
 - It's important for you respond quickly to the attack and its consequences. The longer you wait, the less likely it is that you can recover the affected data.
 
-## Step 1: Verify your backups
+## Step 1: Verify your third-party backups
+
+The Microsoft 365 compliance suite of tools are designed to meet your organizations [legal and regulatory standards](/microsoft-365/compliance/) which is why as a best practice, we [recommend](https://www.microsoft.com/servicesagreement) that you regularly backup your Microsoft 365 Content and Data using Third-Party Apps and Services.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/recover-from-ransomware.md]:
```suggestion
The Microsoft 365 compliance suite of tools is designed to meet your organization's [legal and regulatory standards](/microsoft-365/compliance/) which is why as a best practice, we [recommend](https://www.microsoft.com/servicesagreement) that you regularly back up your Microsoft 365 content and data using third-party apps and services.
```

File: microsoft-365/security/office-365-security/recover-from-ransomware.md [hunk 3]
```diff
@@ -39,9 +40,13 @@ The steps in this article will give you the best chance to recover data and stop
...
+
+In order to ensure you have the ability to restore your data, it's important to regularly test your backups and validate they can be restored in the event of an attack. In the case of an actual attack, these restorations should not be completed until **after** you've removed the ransomware payload (malware) from your environment and **after** you have verified there is no unauthorized access in your Microsoft 365 environments.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/recover-from-ransomware.md]:
```suggestion
In order to ensure you have the ability to restore your data, it is important to regularly test your backups and validate they can be restored in the event of an attack. In the case of an actual attack, these restorations should not be completed until **after** you've removed the ransomware payload (malware) from your environment and **after** you have verified there is no unauthorized access in your Microsoft 365 environments.
```

File: microsoft-365/security/office-365-security/recover-from-ransomware.md [hunk 4]
```diff
@@ -39,9 +40,13 @@ The steps in this article will give you the best chance to recover data and stop
...
 
-If you have offline backups, you can probably restore the encrypted data **after** you've removed the ransomware payload (malware) from your environment.
+To ensure your backups were not also affected by ransomware, the third-party service should be air gapped from your environments.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/recover-from-ransomware.md]:
```suggestion
To ensure your backups were not also affected by ransomware, the third-party service should be air-gapped from your environments.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @drew-russell,  kindly commit my suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by drew-russell** [comment]
@JohanFreelancer9 appreciate the quick eye on here! All changes have been committed.

**Comment by yogkumgit** [comment]
@chrisda Please merge this PR. Thanks

**Pull request merged**